project/frontend/src/components/Loading.js

```diff
@@ -0,0 +1,17 @@
+import './Loading.css';
+
+import React from 'react';
+
+/**
+ * An animated loading icon.
+ */
+function Loading(props) {
+  return (
+    <div id='loading-comments' className='loading-ripple'>
+      <div></div>
+      <div></div>
```

Comment:
What are these divs for?

Reply:
The loading-ripple class targets the child divs in css and creates a little loading animation inside them.

Reply:
Removed the unnecessary id tag on the parent div.